graphql-kotlin-schema-generator/src/test/kotlin/com/expediagroup/graphql/generator/extensions/KFunctionExtensionsKtTest.kt

```diff
@@ -57,8 +57,10 @@ internal class KFunctionExtensionsKtTest {
 
         fun ignored(@GraphQLIgnore ignoredArg: String, notIgnored: String) = "$ignoredArg and $notIgnored"
 
-        fun context(@GraphQLContext contextArg: String, notContext: String) = "$contextArg and $notContext"
+        fun context(contextAnnation: TestContext, contextClass: GraphQLContext, notContext: String) = "Value was $notContext"
```

Comment:
is this still valid? I think we should drop the first argument and update second to `TestContext`